### ISSUE TITLE
chore: Create a runbook for troubleshooting agent issues

### ISSUE THREAD

**Issue by kakkoyun** [parca-dev/parca-agent#442]
e. g: Check Kernel stack capabilities

`sudo bpftrace -e 'profile:ms:20 { @[kstack] = count(); } interval:s:5 { exit(); }'`

cc @javierhonduco @v-thakkar @brancz

**Comment by metalmatze** [comment]
Can we somehow run these test in two different modes?

1. Run these tests when starting the agent
1. Have a CLI command to one off run the same tests if someone wants to run them once (debug container or in a Kubernetes job etc)

**Comment by kakkoyun** [comment]
Yes, definitely. We have another issue concerning start-up checks. #337 
The CLI commands are meant to help users with additional links and guidance.
